src/downloader-resolver.ts

```diff
@@ -0,0 +1,10 @@
+import { Downloader } from './Downloader';
+
+export async function getDownloaderForSystem(): Promise<Downloader<any>> {
+  // TODO: Resolve the downloader or default to GotDownloader
+  // Current thoughts are a dot-file traversal for something like
+  // ".electron.downloader" which would be a text file with the name of the
+  // npm module to import() and use as the downloader
```

Comment:
Is the TODO meant to be resolved before the first release?

Reply:
Not required no, it's a Some Point In The Future TODO